### PR TITLE
chore: update `package.xml` and fix `CMakeLists.txt`

### PR DESCRIPTION
**Pull request by youtalk**
## Description

This PR has updated `package.xml` and `CMakeLists.txt` for each package to prepare releasing from ROS build farm.

## Tests performed

Not applicable.

## Effects on system behavior

Not applicable.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by youtalk** [comment]
`clang-tidy-differential` is still being broken. I'll try to fix it on the other PR.
https://github.com/autowarefoundation/autoware_msgs/actions/runs/9028098716/job/24808131513?pr=91

**Review by isamu-takagi — APPROVED** [review]
LGTM

**Review by mitsudome-r — APPROVED** [review]

**Review by yukkysaito — APPROVED** [review]
LGTM

File: autoware_vehicle_msgs/package.xml [hunk 1]
```diff
@@ -3,9 +3,10 @@
 <package format="3">
   <name>autoware_vehicle_msgs</name>
   <version>1.0.0</version>
-  <description>Interfaces between core Autoware.Auto vehicle components</description>
+  <description>Interfaces between core Autoware vehicle components</description>
   <maintainer email="beginning.fan@autocore.ai">Apex.AI, Inc.</maintainer>
```
Comment (xmfcx) [inline autoware_vehicle_msgs/package.xml:7]: @beginningfan You might want to update this line. (In another PR)

**Pull request merged**